bin/generator/src/main.cpp

```diff
@@ -0,0 +1,85 @@
+#include <boost/json.hpp>
+#include <boost/json/src.hpp>
+#include <boost/program_options.hpp>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "generator.h"
+#include "zkevm_framework/data_types/block.hpp"
+
+int main(int argc, char* argv[]) {
+    boost::program_options::options_description options_desc("Block generator");
+
+    // clang-format off
+    options_desc.add_options()("help,h", "Display help message")
+            ("version,v", "Display version")
+            ("input-file,i", boost::program_options::value<std::string>(), "Input block in json format")
```

Comment:
`config file`, configuration of the block in json format

Reply:
Fixed